scripts/core/editor3/reducers/highlights/store.jsx

```diff
@@ -43,12 +43,26 @@ export const addHighlight = (editorState, selection, data) => {
 };
 
 /**
+ * @description updateHighlight attempts to find the highlight in the passed selection
+ * and update it with the given data
+ * @param {EditorState} editorState
+ * @param {Highlight} highlight 'selection' to update and 'data' to update with.
+ * @returns {EditorState}
+ */
+export const updateHighlight = (editorState, {selection, data}) => {
+    const m = getHighlights(editorState.getCurrentContent());
```

Comment:
imo m/k could use different names

Reply:
It's **m**ap and **k**ey but I will happily change it.